.gitignore

```diff
@@ -60,6 +60,7 @@ typings/
 # next.js build output
 .next
 
-
 .DS_Store
 
+.eslintrc
```

Comment:
We want this to be in the repo so that styling is consistent across contributors